Documentation/api-reference.md

```diff
@@ -0,0 +1,1284 @@
+# Adobe Experience Platform Media for Edge Network Extension - iOS
+
+## Prerequisites
+
+To set up the extension and start using these APIs, see the [Getting Started Guide](getting-started.md).
+
+## API reference
+
+| APIs                                                  |
+| ----------------------------------------------------- |
+| [extensionVersion](#extensionVersion)                 |
+| [registerExtension](#registerExtension)               |
+| [createTracker](#createTracker)                       |
+| [createTrackerWithConfig](#createTrackerWithConfig)   |
+| [createMediaObjectWith](#createMediaObjectWith)               |
+| [createAdBreakObjectWith](#createAdBreakObjectWith)           |
+| [createAdObjectWith](#createAdObjectWith)                     |
+| [createChapterObjectWith](#createChapterObjectWith)           |
+| [createQoEObjectWith](#createQoEObjectWith)                   |
+| [createStateObjectWith](#createStateObjectWith)               |
+
+## Media Tracker API reference
+
+| APIs                                                  |
+| ----------------------------------------------------- |
+| [trackSessionStart](#trackSessionStart)               |
+| [trackPlay](#trackPlay)                               |
+| [trackPause](#trackPause)                             |
+| [trackComplete](#trackComplete)                       |
+| [trackSessionEnd](#trackSessionEnd)                   |
+| [trackError](#trackError)                             |
+| [trackEvent](#trackEvent)                             |
+| [updateCurrentPlayhead](#updateCurrentPlayhead)       |
+| [updateQoEObject](#updateQoEObject)                   |
+
+------
+
+### extensionVersion
+
+The extensionVersion API returns the version of the Media for Edge Network extension.
+
+#### Swift
+
+##### Syntax
+```swift
+static var extensionVersion: String
+```
+
+##### Example
+```swift
+let extensionVersion = EdgeMedia.extensionVersion
+```
+
+#### Objective-C
+
+##### Syntax
+```objectivec
++ (nonnull NSString*) extensionVersion;
+```
+
+##### Example
+```objectivec
+NSString *extensionVersion = [AEPMobileEdgeMedia extensionVersion];
+```
+------
+
+### registerExtension
+
+Registers the Media for Edge Network extension with the Mobile Core extension.
+
+The extension registration occurs by passing the Media for Edge Network extension to the [MobileCore.registerExtensions](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/mobile-core-api-reference#registerextension-s) API.
```

Comment:
This should point to the Adobe Doc page instead of the older GitBook page: https://developer.adobe.com/client-sdks/documentation/mobile-core/api-reference/#registerextensions.